conf/application.conf

```diff
@@ -69,34 +105,19 @@ auditing {
   }
 }
 
-controllers {
-  controllers.Assets = {
-    needsAuditing = false
-  }
-
-  uk.gov.hmrc.govukfrontend.controllers.Assets = {
-    needsAuditing = false
-  }
-
-  uk.gov.hmrc.hmrcfrontend.controllers.Assets = {
-      needsAuditing = false
-  }
+google-analytics {
+  token = N/A
+  host = auto
 }
 
-play.i18n.langCookieHttpOnly: "true"
-
-# Change this value to true to enable Welsh translations to be loaded from messages.cy, and to display the language toggle
-features.welsh-language-support = false
-
-# Replace play.i18n.langs with the commented out line below when your service has been fully translated into Welsh
-# to enable Welsh translations for all content, including the standard headers and footers.
-# play.i18n.langs = ["en", "cy"]
-play.i18n.langs = ["en"]
-
-# To integrate with tracking-consent-frontend, uncomment and update the
-# gtm.container property below to be one of a, b, c, d, e, f or sdes
-#tracking-consent-frontend {
-#  gtm.container = "<INSERT YOUR GTM CONTAINER LETTER HERE>"
-#}
+assets {
```

Comment:
I think this is no longer required as we are using play-frontend-hmrc. You could check with Martin.